docs/feature-flags/creating-feature-flags/trigger.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: Triggers
+# sidebar_position:
+slug: /feature-flags/creating-feature-flags/triggers
+description: Use the flag triggers to automatically turn feature flags on or off by calling a URL using the POST method.
+tags: ['guide', 'automation', 'trigger', 'feature-flag']
+---
+
+import CenteredImg from '@site/src/components/centered-img/CenteredImg';
+
+Flag triggers are a feature that Bucketeer provides to let you change the flag's state (ON/OFF) by calling a URL. As a result, you can use this option to enable or disable flags during CI/CD (Continuous Integration/Continuous Deployment) pipelines or based on an APM (Application Performance Monitoring) integration.
+
+To access the trigger page on the Bucketeer dashboard, select the **Feature Flags** tab, choose the desired flag, and click on its name. Select the **Trigger** tab at the top of the new page. You can check the existing triggers related to the current feature flag.
```

Comment:
I think we can simplify this `To access the trigger page on the Bucketeer dashboard, select the **Feature Flags** tab,` by just saying `On the Feature Flag details page`.

Also, `Select the **Trigger** tab at the top of the new page` is not a new page.